ui/src/tool-panel/location-track/location-track-infobox.tsx

```diff
@@ -162,7 +162,12 @@ const LocationTrackInfobox: React.FC<LocationTrackInfoboxProps> = ({
                 id={locationTrack.id}
                 layoutContext={layoutContext}
                 showLinkedSwitchesRelinkingDialog={() => setConfirmingSwitchRelinking(true)}
-                editingDisabled={editingDisabled}
+                switchRelinkingDisabled={editingDisabled || locationTrack.state === 'DELETED'}
+                switchRelinkingDisabledMessageKey={
+                    locationTrack.state === 'DELETED'
+                        ? 'tool-panel.location-track.switches-for-deleted-track-cannot-be-relinked'
```

Comment:
Oletan että tässä on näin päin koska tuo ylempi on yhdistelmä mutta tuntuisi mielekkäämmältä että  tuonne kannattaisi passata editingDisabled ja state ja antaa komponentin itse päätellä että relinking on myös disabled noiden seurauksena + tuntea tuon (ei dynaamisen) lokalisaatioavaimen.